### PR TITLE
fix the TELEGRAM_ADMIN env variable to support multi values

### PR DESCRIPTION
**Pull request by krasi-georgiev**
Signed-off-by: Krasi Georgiev <8903888+krasi-georgiev@users.noreply.github.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,8 +5,6 @@ RUN apk add --update ca-certificates tini
 COPY ./default.tmpl /templates/default.tmpl
 COPY ./alertmanager-bot /usr/bin/alertmanager-bot
 
-USER nobody
```
Comment (krasi-georgiev) [inline Dockerfile:8]: not sure why this was added, but with this the folder inside can only be written with root so removing this allows running the cli as root to have write access

**Comment by metalmatze** [comment]
Oh hi Krasi! :wave: :blush: 
The bot already works with multiple admins :)
See https://github.com/metalmatze/alertmanager-bot/issues/127#issuecomment-689631184

However, given that quite a number of people have stumpled upon this, I'm inclined to change it for a comma separate list, as this is way more intuitive.
Do you think this is a breaking change?

**Comment by krasi-georgiev** [comment]
It doesn't work when using env var.

**Comment by krasi-georgiev** [comment]
als in bash you can't set the env var with dots like `TELEGRAM.ADMIN` so needs to be provided as `TELEGRAM_ADMIN`

**Pull request closed without merging**